### PR TITLE
MRG Epoch selection example

### LINKED ISSUE #5117: epochs[['foo', 'bar']] bug
Let's say you have an epochs object which you believe has labels 'foo' and 'bar', but really 'bar' doesn't exist as a label in the object.

epochs['bar'].average() throws an error, which seems reasonable

epochs[['foo', 'bar']].average() does not throw an error while it should.

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
Closes #5117

### CODE REVIEW

File: tutorials/plot_object_epochs.py [hunk 1]
```diff
@@ -75,6 +75,14 @@
 print(epochs[1:5])
 print(epochs['Auditory/Right'])
 
+###############################################################################
+# Note the '/'s in the event code labels. These separators allow tag-based
+# selection of epoch sets; every string separated by '/' can be entered, and
+# returns the subset of epochs matching any of the strings. E.g.,
+
+print(epochs['Right'])
+print(epochs['Right', 'Left'])
```
[the remark this answers is not in the document]
Comment (jona-sassenhagen) [reply tutorials/plot_object_epochs.py]: Sure.

File: tutorials/plot_object_epochs.py [hunk 2]
```diff
@@ -75,6 +75,23 @@
...
+
+
+###############################################################################
+# Note that MNE will not complain if you ask for tags not present in the object,
+# as long as it can find some match.
```
Comment (larsoner) [inline tutorials/plot_object_epochs.py]: ", as the ['Right', 'Left'] below is treated as "Right" or "Left", and the object does have "Right"-type epochs."

or something like this.

You could even put back the one that raises an error (`'Left'`), just put it in a try/except. It seems worth keeping to point out that if none can be found you do get an error.

**Comment by jona-sassenhagen** [comment]
Like this?

**Comment by larsoner** [comment]
CircleCI for other reviewers:

https://8786-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_object_epochs.html

**Comment by larsoner** [comment]
PEP8

https://travis-ci.org/mne-tools/mne-python/jobs/412181073#L3242

**Pull request merged**